### DISCUSSION TITLE
No input-device found, insert one to fly

### DISCUSSION THREAD

**Discussion by HerrnWang** [bitcraze/discussions#1127] [Q&A] [answered]
The OS is windows10. And I use Pycharm. 
cfclient can't find the input device, but when i move the crazyflie, the cfclient will show some changes.
![屏幕截图 2024-01-12 200411](https://github.com/bitcraze/discussions/assets/141895001/ad52704e-dedf-4c28-a393-fd4c0b090c3e)
but after I move the crazyflie, it looks like this:
![屏幕截图 2024-01-12 200553](https://github.com/bitcraze/discussions/assets/141895001/43e80db8-b5e1-43ef-b0eb-97922660d12a)
In Pycharm's Terminal it shows:
![屏幕截图 2024-01-12 200927](https://github.com/bitcraze/discussions/assets/141895001/5efed41b-5448-4d70-b63d-fdb8a77eef25)

the zadig drive is also installled. Situation above is normal？or there are some problems?

**Comment by knmcguire** [comment, marked as answer]
Hi! Yeah that is a common problem on Windows.. but it is just a warning and in terms of functionality nothing is really different. I think this is mostly a difference between the pysdl2 library for linux and windows that it keeps searching for USB devices... It's annoying but it is considered 'normal behavior'.

We actually never had an issue for this so I just made one! https://github.com/bitcraze/crazyflie-clients-python/issues/693

**Reply by HerrnWang** [reply]
vielen Dank
